Haskell/Haskell.sublime-syntax

```diff
@@ -1254,6 +1255,19 @@ contexts:
       scope: meta.quoted.quasi.haskell punctuation.section.quoted.end.haskell
       pop: 1
 
+  hsx-quotes:
+    - match: (\[)(hsx)(\|)
+      scope: meta.quoted.quasi.haskell
```

Comment:
I presume you didn't chose `meta.string` here on purpose? Do we know of any other occurances of `meta.quoted` in the current code base?

Reply:
> Quasiquoting allows programmers to use custom, domain-specific syntax to construct fragments of their program. Along with Haskell's existing support for domain specific languages, you are now free to use new syntactic forms for your EDSLs. 
> see: https://wiki.haskell.org/Quasiquotation

That description doesn't read like quasi quotes are plain strings. They rather seem to have a special meaning. Hence special meta scope was choosen when rewriting Haskell and so is for hsx.